virtcontainers/qemu.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	govmmQemu "github.com/intel/govmm/qemu"
 	"github.com/opencontainers/selinux/go-selinux/label"
-	"github.com/opentracing/opentracing-go"
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
uhm this looks odd

Reply:
Looks changed by my IDE go formatter. will change it back.

Reply:
Fixed